main.go

```diff
@@ -130,13 +133,15 @@ func main() {
 	// and we want to loosen some assumptions, eg. doesn't matter if not paired.
 	vehicleInfo, err := blockingGetVehicleInfo(logger, ethAddr, lss, env)
 	if err != nil {
-		logger.Fatal().Err(err).Msgf("cannot start edge-network because no on-chain pairing was found for this device addr")
+		logger.Fatal().Err(err).Msgf("cannot start edge-network because no on-chain pairing was found for this device addr: %s", ethAddr.Hex())
```

Comment:
i'm thinking this one we should send

Reply:
all Fatal errors automatically send to mqtt